lib/cisco_node_utils/bgp_neighbor.rb

```diff
@@ -503,5 +503,25 @@ def mode_cli_to_symbol(cli)
         fail KeyError
       end
     end
+
+    def default_peer_type
+      config_get_default('bgp_neighbor', 'peer_type')
+    end
+
+    def peer_type=(val)
+      if val == default_peer_type
+        @set_args[:state] = 'no'
+      else
+        Feature.nv_overlay_evpn_enable unless
+          Feature.nv_overlay_evpn_enabled?
```

Comment:
You don't actually need the `unless Feature.nv_overlay_evpn_enabled?` bit.  The `Feature.nv_overlay_evpn_enable` takes care of this.

https://github.com/cisco/cisco-network-node-utils/blob/develop/lib/cisco_node_utils/feature.rb#L193

Reply:
Removed the check